vendor/knative.dev/eventing/pkg/apis/feature/features.go

```diff
@@ -66,6 +87,10 @@ func NewFlagsConfigFromMap(data map[string]string) (Flags, error) {
 			flags[sanitizedKey] = Disabled
 		} else if strings.EqualFold(v, string(Enabled)) {
 			flags[sanitizedKey] = Enabled
+		} else if strings.EqualFold(v, string(Permissive)) {
```

Comment:
nit: consider converting this to a case since we are not using utf-8 edge cases, using toUpper should be enough (I think)